.github/npm-publish.yml

```diff
@@ -0,0 +1,36 @@
+name: NPM Registry Publication
+
+on:
+  push:
+    tags:
+      - node-[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: node
+
+    strategy:
+      matrix:
+        node-version:
+         - 16.x
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Build the distribution (from TypeScript to JavaScript)
+        run: npm run build
+
+      - name: Publish the package
+        run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
I think Hiro's NPM_TOKEN is set already in the `scalar-labs` group on GitHub.
So no need to set up any other settings on GitHub.

Reply:
It's set in the `scalardl-node-client-sdk` repo and it's not set in the organization secret. 
So, I think I need to set it to this repo, as well. Can you give me the one?

Reply:
I am sorry. It's my mistake.
I thought NPM_TOKEN has been set as an organization secret already but it is CR_PAT.

> Can you give me the one?
This package will be published as a scoped package to the @scalar-labs organization in the NPM registry. The NPM_TOKEN will be your (feeblefakie) NPM access token.

Please check this document https://docs.npmjs.com/creating-and-viewing-access-tokens
(it seems the access token can be seen only once right after created though)